nginx-prod.conf

```
@@ -112,16 +112,31 @@ http {
             return 404;
         }
 
+        #redirect informationmodels to information-models
+        location ~* ^/informationmodels(.*) {
+            return 301 /information-models$1;
```

Du tar med schema og host for `return 303 ...` andre steder, må du gjøre det her og?
```suggestion
            return 301 https://data.norge.no/information-models$1;
```